resource_types/core/energy/102-generation/energygeneration.oas.yaml

```diff
@@ -18,7 +18,7 @@ produces:
   #- application/vnd.ocf+cbor
   - application/json
 paths:
-  "/ein":  # suggested URI
```

Comment:
After thinking about this again, I realized I was still thinking about the convention wrong. Every device can just report the energy in and out. It's weird to report "this is the energy generated to me". So I'm reverting my request to change -- I think it's fine as `ein/eout`. Sorry for the confusion.

Reply:
Also the `st` enum example still isn't a complete list?

Reply:
Highlighting that changing back means that a solar panel would *not* host this resource (at least, the URI doesn't make as much sense then). So this is implying that the resource is hosted on a device that measures its *own* input, and energy generation devices don't report via this resource.

Reply:
Yes I understand that now! As long as we're consistent I'm fine -- and I'm sure we'll learn what's most useful when we get feedback from manufacturers.